Game.js

```diff
@@ -0,0 +1,72 @@
+import Board from "./Board.js";
+import ComputerPlayer from "./ComputerPlayer.js";
+//const HumanPlayer = require("./HumanPlayer.js")
+//const ComputerPlayer = require("./ComputerPlayer")
+//const Board = require("./Board.js")
+class Game {
+  constructor(player) {
+    this.player = player;
+    this.ComputerPlayer = ComputerPlayer;
+    this.board = new Board
```

Comment:
having `this.board = new Board(this.ComputerPlayer.secretWordLength());` revolved an error that your code was throwing